pkgs/by-name/zs/zsh-zhooks/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}: let
+  pname = "zhooks";
+  install_path = "share/zsh/${pname}";
+in
+  stdenv.mkDerivation {
+    pname = "zsh-${pname}";
```

```suggestion
    pname = "zsh-zhooks";
```
declaring pname twice can easily lead to errors. Lets just write them out

---

good catch thanks, fixed